engine/admin/volumes/tmpfs.md

```diff
@@ -0,0 +1,100 @@
+---
+description: Using tmpfs mounts
+title: Use tmpfs mounts
+keywords: storage, persistence, data persistence, tmpfs
+---
+
+[Volumes](volumes.md) and [bind mounts](bind-mounts.md) are mounted into the
+container's filesystem by default, and their contents are stored on on the host
+machine.
+
+There may be cases where you do not want to store a container's data on the host
+machine, but you also don't want to write the data into the container's writable
+layer, for performance or security reasons, or if the data relates to
+non-persistent application state.
+
+To give the container access to the data without writing it anywhere
+permanently, you can use a `tmpfs` mount, which is only stored in the host
+machine's memory (or swap, if memory is low). When the container stops, the
+`tmpfs` mount is removed. If a container is committed, the `tmpfs` mount is not
+saved.
+
+## Limitations of tmpfs containers
+
+- `tmpfs` mounts cannot be shared among containers.
+- `tmpfs` mounts only work on Linux containers, and not on Windows containers.
+
+## Use a tmpfs mount in a container
+
+To use a `tmpfs` mount in a container, use the `--tmpfs` flag, or use the
+`--mount` flag with `type=tmpfs` and `destination` options. There is no
+`source` for `tmpfs` mounts. The following example creates a `tmpfs` mount at
+`/app` in a Nginx container. The first example uses the `--tmpfs` flag and the
+second uses the `--mount` flag.
+
+```bash
```

Comment:
I know I'm being boring by now but can we switch the order of these examples to emphasize the use of `--mount`?